include/cpr/auth.h

```diff
@@ -9,18 +9,14 @@ namespace cpr {
 
 class Authentication {
   public:
-    Authentication(std::string&& username, std::string&& password)
-            : username_(std::move(username)),
-              password_(std::move(password)), auth_string_{this->username_ + ":" +
-                                                           this->password_} {}
+    Authentication(const std::string& username, const std::string& password)
+            : auth_string_{username + ":" + password} {}
     virtual ~Authentication() = default;
 
     const char* GetAuthString() const noexcept;
 
   private:
```

Comment:
Could you change the visibility to protected for properties.

Reply:
Changed.